src/docs/style/color.mdx

```diff
@@ -1,12 +1,14 @@
-import { ColorPalette, HeadlineWithToggle } from '../../app/components';
+import { ColorPalette, HeadlineWithToggle, MaterialDesignDescription } from '../../app/components';
 import * as Colors from '@pxblue/colors';
 import * as BrandingColors from '@pxblue/colors-branding';
 import { Typography } from '@material-ui/core';
 import { Bookmark } from '@material-ui/icons';
 
 <HeadlineWithToggle />
 
-PX Blue offers a variety of different colors for use in your applications. Our color palettes utilize a weighted approach to give designers and developers a versatile set of colors for solving common color-related issues (e.g., accessibility). [Learn more about weighted color palettes](https://material.io/design/color/).
+PX Blue offers a variety of different colors for use in your applications. Our color palettes utilize a weighted approach to give designers and developers a versatile set of colors for solving common color-related issues (e.g., accessibility).
+
+<MaterialDesignDescription title={'The color system'} url={'https://material.io/design/color/'} />
```

Comment:
Is this one a design pattern or just information about color systems? Should it have a non-default description?